internal/handlers/irc/handlers.go

```diff
@@ -6,6 +6,11 @@ import (
 	"github.com/lrstanley/girc"
 )
 
+const (
+	joinFmt = "<system> %s joins"
+	partFmt = "<system> %s parts"
```

Comment:
It's a stylistic preference, but I assume this is sent in Telegram when an IRC user parts/joins:

`<system> jwflory joins`

My preference would be something like:

`* jwflory joins`

It's my preference, but I won't make this suggestion blocking.